utility-scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Utility-scripts Info
```

Comment:
Is this still necessary since we are moving away from Scrutiny and just using the .txt in this repo as our starting point?